src/OrchardCore.Modules/OrchardCore.AdminDashboard/Drivers/DashboardContentDisplayDriver.cs

```diff
@@ -0,0 +1,119 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using OrchardCore.ContentManagement;
+using OrchardCore.ContentManagement.Display.ContentDisplay;
+using OrchardCore.Contents;
+using OrchardCore.DisplayManagement.Handlers;
+using OrchardCore.DisplayManagement.Views;
+using OrchardCore.ContentManagement.Display.ViewModels;
+
+
+namespace OrchardCore.AdminDashboard.Drivers
+{
+    public class DashboardContentDisplayDriver : ContentDisplayDriver
+    {
+        private readonly IHttpContextAccessor _httpContextAccessor;
+        private readonly IAuthorizationService _authorizationService;
+        private readonly IContentManager _contentManager;
+
+
+        public DashboardContentDisplayDriver(IHttpContextAccessor httpContextAccessor,
+            IAuthorizationService authorizationService,
+            IContentManager contentManager)
+        {
+            _httpContextAccessor = httpContextAccessor;
+            _authorizationService = authorizationService;
+            _contentManager = contentManager;
+
+        }
+
+        public override async Task<IDisplayResult> DisplayAsync(ContentItem model, BuildDisplayContext context)
+        {
+            var httpContext = _httpContextAccessor.HttpContext;
+            var isManageRequest = httpContext.Items.ContainsKey(typeof(DashboardManageAttribute));
+            var hasPublished = await _contentManager.HasPublishedVersionAsync(model);
```

Comment:
Here I think you can just return `Task.FromResult<IDisplayResult>(null)` if the feature is not set.

Reply:
Done